check_upgrade/check_upgrade_test.go

```diff
@@ -26,6 +27,9 @@ import (
 )
 
 func TestCheckUpgrade(t *testing.T) {
+	if runtime.GOOS != "linux" {
```

Comment:
Maybe we need to find out ways to change it for other os?

Reply:
I forgot about the DGRAPH_BINARY env variable... which can allow these tests to run... I updated all these to also check for that envvar.